source/preprocessor.tex

```diff
@@ -983,6 +1013,29 @@
 arguments that would otherwise act as preprocessing directives,\footnote{A \grammarterm{conditionally-supported-directive} is a preprocessing directive regardless of whether the implementation supports it.}
 the behavior is undefined.
 
+\pnum
+\begin{example}
+The following defines a function-like
+macro whose value is the maximum of its arguments.
+It has the advantages of working for any compatible types of the arguments
+and of generating in-line code without the overhead of function calling.
```

Comment:
It's not clear what these are advantages compared to. Maybe we could just strike this sentence, and describe the macro form of `max` has having only disadvantages?

Reply:
Agreed. Removed.